src/styles/mixins.scss

```
@@ -0,0 +1,12 @@
+@mixin cell {
+  width: $cell-size;
+  height: $cell-size;
+  border: 1px solid $border-color;
+  background-color: $background-color;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  position: relative;
+  cursor: pointer;
+  transition: transform 0.5s;
+}
```

There is no need for such a mixin, put all __day properties in a standard way.

If you want to use a mixin, let's do it in more reasonable case:
```suggestion
@mixin dimensions {
  width: $cell-size;
  height: $cell-size;
}
```